odl/solvers/functional/default_functionals.py

```diff
@@ -2292,6 +2293,82 @@ def gradient(self):
                 (1 / self.sigma) * self.functional.proximal(self.sigma))
 
 
+class HuberL1L2(Functional):
```

Comment:
Adding this partially solves #597 which should be mentioned i guess

Reply:
Also not 100% sure about the naming, is there no way to create something like `Huber` and control the `L1L2` part via parameters?

Reply:
Good point, maybe this is possible. So far, I only came across the L1L2 version. Maybe we can add a TODO note and implement this when there is need for this (which may be never?!?)?

Reply:
I still feel that we need to improve the naming here before we go ahead. Is there any "reasonable" way to make this simply `Huber`?